Formula/s/screenpipe.rb

```diff
@@ -30,13 +32,16 @@ class Screenpipe < Formula
   end
 
   def install
+    system "curl", "-fsSL", "https://bun.sh/install", "|", "bash"
```

Comment:
That's very much disallowed:

https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things